extensions/data-plane/data-plane-http/src/test/java/org/eclipse/edc/connector/dataplane/http/pipeline/HttpRequestParamsTests.java

```diff
@@ -77,7 +76,7 @@ void verifyHeaders() {
     @Test
     void verifyComplexUrl() {
         var path = "testpath";
-        var queryParams = "test-queryparams";
+        var queryParams = "foo=bar&hello=world";
```

Comment:
since we're actually parsing the query params, it may make sense to test the negative case here too, i.e. an invalid queryparams string, e.g. `foo-bar&hello`

Reply:
Done in `HttpSourceRequestParamsSupplier` in which I moved the query params parsing logic.